schemas/spacefile/spacefile.v0.schema.json

```diff
@@ -118,25 +114,42 @@
           "description": "Routes that will be available publicly",
           "type": "array",
           "minItems": 1,
+          "uniqueItems": true,
```

Comment:
I don't think that this work for object type.

Reply:
These items are strings.